app/channel.go

```diff
@@ -340,15 +340,19 @@ func (a *App) createDirectChannel(userId string, otherUserId string) (*model.Cha
 		close(uc2)
 	}()
 
-	if result := <-uc1; result.Err != nil {
+	result := <-uc1
+	if result.Err != nil {
 		return nil, model.NewAppError("CreateDirectChannel", "api.channel.create_direct_channel.invalid_user.app_error", nil, userId, http.StatusBadRequest)
```

Comment:
Did you intend to leave the details parameter as `userId` or were you intending to add `result.Err.Error()`?

Reply:
I wasn't changing anything there, so I was just leaving it as was before.

Reply:
Ok, just seems weird but not related to this PR.